CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Version 1.0.0-alpha1 (Apr 13, 2024)
+## Version 1.0.0
```

Comment:
I'm assuming we deliberately chose to mix new entries into the changelog for alpha1 to create a consolidated changelog for the final 1.0 release, so I updated the header accordingly.